pkg/ledger/process_test.go

```diff
@@ -309,6 +309,7 @@ func TestLedger_processTx(t *testing.T) {
 				Transaction: core.Transaction{
 					TransactionData: core.TransactionData{
 						Timestamp: now,
+						Postings:  []core.Posting{{}},
```

Comment:
I don't think this should be necessary, can be avoided in the `insertTransaction` switch case

Reply:
I'm doing it now